enterprise/cmd/frontend/internal/authz/resolvers/resolver.go

```
@@ -544,6 +552,91 @@ func (r *permissionsInfoResolver) Unrestricted() bool {
 	return r.unrestricted
 }
 
+var permissionsInfoRepositoryConnectionMaxPageSize = 100
+
+var permissionsInfoRepositoryConnectionOptions = &graphqlutil.ConnectionResolverOptions{
+	OrderBy:     database.OrderBy{{Field: "repo.name"}},
+	Ascending:   true,
+	MaxPageSize: &permissionsInfoRepositoryConnectionMaxPageSize,
+}
+
+func (r *permissionsInfoResolver) Repositories(ctx context.Context, args graphqlbackend.PermissionsInfoRepositoriesArgs) (*graphqlutil.ConnectionResolver[graphqlbackend.PermissionsInfoRepositoryResolver], error) {
+	if r.userID == 0 {
+		return nil, nil
+	}
```

Should we return an error instead saying userID is invalid?